HHS/Head-Start-TTADP
# [ Prod ] Automate CDI grants to recipients mapping;

## Description of change

Automatically associated CDI grants with the recipient the CDI grant replaced.
Fixed the FEI pseudo goals.
Fix bug with date display within unchanged reports

## How to test
- Verify the site works.

### Testing the date error
- New report. No need to touch anything other than start or end date.
- Enter 10/04/2023 for start date (or use calendar)
- Enter 10/20/2023 for end date (or use calendar)
- Save and continue
- Without touching anything else, use the side navigation (activity summary in progress) to go back to the activity summary
- Change means that you see the date you entered

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-814
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2092
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2203
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2217

## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy


### After merge/deploy
